public_html/assets/css/nf-core.css

```diff
@@ -906,6 +923,25 @@ Based on https://codepen.io/wintr/pen/beBJBb */
   padding: 1rem 1rem 0.5rem;
 }
 
+.fa_icon_picker.popover {
+  width: 315px;
```

Comment:
why in `px`? don't the icons scale with the rest of the fonts, so use `rem` instead?

Reply:
Yeah, probably should do. Got to the value by tweaking in the browser so was being lazy. I'll fix 👍

Reply:
Ok, switched to `rem` 👍